docs/source/release_notes.rst

```diff
@@ -6,6 +6,50 @@
 Releases
 ========
 
+Version 1.3.0-beta.2
+====================
+
+What's New
+----------
+
+* Modules
+
+  * Utility ``better_arg_parser`` - supports relative paths and removes the
+    choice case sensitivity.
+  * ``zos_operator`` - supports new options **wait** and **wait_time_s** such
+    that you can specify that ``zos_operator`` wait the full **wait_time_s** or
+    return as soon as the first operator command executes.
+
+* Bug Fixes
+
+  * Modules
+
+    * Module ``zos_job_submit`` was updated to remove all trailing **\r** from
+      jobs that are submitted from the controller.
+    * Module ``zos_copy`` was updated to support copying data set members that
+      are program objects to a PDSE. Prior to this update would yield an error:
+      **FSUM8976 Error writing <src_data_set_member> to PDSE member
+      <dest_data_set_member>: EDC5000I No error occurred**
+    * Job module utility used by several modules when performing job related
+      reading operations would fail when encountering a special character. The
+      utility has been updated to use a custom written parsing routine capable
+      of handling special characters.
+
```

Comment:
Suggested rewrite :

Job module utility has been updated to use a custom written parsing routine capable of handling special characters to prevent job related reading operations from failing when a special character is encountered.